src/Discovery/src/ConsulBase/Steeltoe.Discovery.ConsulBase.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging" Version="$(ExtensionsVersion)" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="$(ExtensionsVersion)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(JsonNetVersion)" />
+    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="$(HttpVersion)" />
```

Comment:
`$(HttpVersion)` = 4.3.3, we may need to bump it also

Reply:
The latest available version of System.Net.Http is 4.3.4. I think I will make a new variable for WinHttpHandler and bump it to latest stable version of 4.7.0